.github/workflows/e2e-main.yml

```
@@ -9,6 +9,7 @@ on:
       - "docs/**"
       - "**.md"
       - ".circleci/**"
+      - "**unit.spec.js"
```

Not entirely sure this glob works.
According to the [glob tester](https://toools.cloud/miscellaneous/glob-tester) it should. But IIRC I was trying it out in my fork and the workflow ran even if the only changed file was a unit test.

---

Another potential issue with this glob is that it doesn't account for `.jsx`, `.`ts` or `.tsx` unit tests.

---

Don't know much about this, but I would probably use
```suggestion
      - "**/*.unit.spec.{js,jsx,ts,tsx}"
```